### ISSUE TITLE
Custom DB MVP 7 - Multiple shards per index

### ISSUE THREAD

**Issue by etiennedi** [semi-technologies/weaviate#1153]
## User Journey

* [ ] The user can select the desired amount of shards
* [ ] All searches happen across the required shards in parallel
* [ ] The calculation of shard identifier from UUID is deterministic (e.g. Murmur Hasher)
* [ ] We are prepared to change the amount of shards in the future (e.g. Ring hashing)
* [ ] A direct lookup goes directly to the identified shard (not a search!)

## Out of scope
* Distributing shards across nodes
* Changing the number of shards

**Comment by etiennedi** [comment]
Dear Mr/Mrs Stale bot, you're a bit too quick to close, MVP 4 is currently in progress. ;-)

**Comment by bobvanluijt** [comment]
![robot](https://cdn2.iconfinder.com/data/icons/robot-22/512/robot_emoticon_emoji_sticker_thumbs_up-512.png "robot")
